OrbitCore/Utils.h

```diff
@@ -110,18 +111,24 @@ inline bool IsAllDigits(const std::string_view value) {
   return std::all_of(value.begin(), value.end(), IsDigit);
 }
 
+namespace OrbitCore {
+namespace OrbitUtils {
```

Comment:
Should we really have this double namespace?

Reply:
This was because I had OrbitCore::Error class at some point in this PR. It is removed now